ManagedCache+CoreDataClass.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  ManagedCache+CoreDataClass.swift
+//  FeedStoreChallenge
+//
+//  Created by kkolontay on 7/13/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+@objc(ManagedCache)
+public class ManagedCache: NSManagedObject {
+	static var typeName: String {
```

Comment:
This method is used only in the scope of this file thus it can be private.
Please make it `private`

```suggestion
	private static var typeName: String {
```

Reply:
done